### PR TITLE
Mik delft

### PR DESCRIPTION
**Pull request by jcolomb**
Added blog post as report from Mik's work in Delft.

- [x] review by Mik
- [x] getting Guy and Robert (and others?) accord
- [x] publish

### CODE REVIEW

File: content/blog/2023-07-28-three-months-in-delft-two-open-hardware-projects/index.en.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
...ensiveness and completeness of the documentation is crucial as it optimizes your work to many different professional backgrounds. Naturally, Delft’s highly diverse community, involving engineers, businesses, biologists, artists, and creative hobbyists fosters an environment for innovation. For instance, I met a developer whom decided to build an anti-theft system to prevent their bike from being stolen. This phone-size device is sensitive to vibrations, which can trigger the alarm to go off. This prompted me to determine if their hardware was also able to detect animal licks for my setup. The same technology is already implemented in some lick-detectors but have high financial cost. Additionally, the Delft community introduced me to a preliminary method for replacing the microstimulation setup’s most expensive component: the micromanipulator. The Open Hardware Automated microscope uses a lower-cost alternative to manipulate a platform, which could be adapted for microstimulation. By making hardware designs and documentation available to a wider audience, open hardware can foster a community of developers and enthusiasts who can work together to improve and enhance hardware systems.
```
Comment (mik-schutte) [inline content/blog/2023-07-28-three-months-in-delft-two-open-hardware-projects/index.en.md:93]: Anti-bike theft: https://www.tudelft.nl/open-hardware/projects/anti-bike-theft-device
Automated microscope: https://www.tudelft.nl/open-hardware/projects/automated-microscope

File: content/blog/2023-07-28-three-months-in-delft-two-open-hardware-projects/index.en.md [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+
+Finally, I learned that open hardware documentation is not just important for hobbyists and enthusiasts. Open hardware documentation can also benefit commercial and industrial applications by enabling faster prototyping, reducing development costs, and facilitating innovation. By making hardware designs and documentation available to a wider audience, companies can tap into a diverse pool of talent and expertise and create more innovative and sustainable hardware products.
+
+{{< /card2 >}}
+
+#  Hardware work
+
+## Berlin time
+
+In November 2021, I started his master project that would use the microstimulation setup. However, the setup was no longer available in the lab. The people who worked on that paradigm had left the lab, too. Thus, I set out to re-establish the setup. My only source of information was an incomplete wiring diagram. Luckily, with the help of colleagues, I was able to get the basics of the setup to work: stimulate an electrode and detect licks. Next, I implemented the behavioural aquisition, meaning that I set up a camera that were able to clearly capture individual whisker movement and a camera that clearly captured the pupil of the animal.
```
Comment (mik-schutte) [inline content/blog/2023-07-28-three-months-in-delft-two-open-hardware-projects/index.en.md:109]: In all honesty, Moritz was there to help a lot and he worked with the paradigm before maybe we should mention him explicitly.

File: content/blog/2023-07-28-three-months-in-delft-two-open-hardware-projects/index.en.md [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+
+
+# Project history
+
+The Microstimulation setup had been developed in the lab mostly by Guy Doron, its use led to two important publication for the lab
+ [(Doron et al., 2020](https://www.science.org/doi/10.1126/science.aaz3136); [Shin et al., 2021)](https://www.science.org/doi/10.1126/science.abk1859).
+ 
+ {{< /card >}}
+
+# Delft Open Hardware Academy
```
Comment (mik-schutte) [inline content/blog/2023-07-28-three-months-in-delft-two-open-hardware-projects/index.en.md:61]: https://hackmd.io/ya-g7omdSI--Vx0JcayB1Q?view

**Review by mik-schutte — CHANGES_REQUESTED** [review]
I like what you have done with the post. Mostly just want to add some links.

File: content/blog/2023-07-28-three-months-in-delft-two-open-hardware-projects/index.en.md (hunk 1 above)
Comment (jcolomb) [reply content/blog/2023-07-28-three-months-in-delft-two-open-hardware-projects/index.en.md:93]:
```suggestion
The existing community of Delft Open Hardware is surrounded by people that are also working on optimizing their documentation for third part use. From this surrounding it is easy to obtain tips and ideas. Moreover, having a community that quickly provides feedback on the comprehensiveness and completeness of the documentation is crucial as it optimizes your work to many different professional backgrounds. Naturally, Delft’s highly diverse community, involving engineers, businesses, biologists, artists, and creative hobbyists fosters an environment for innovation. For instance, I met a developer whom decided to build an anti-theft system to [prevent their bike from being stolen](https://www.tudelft.nl/open-hardware/projects/anti-bike-theft-device). This phone-size device is sensitive to vibrations, which can trigger the alarm to go off. This prompted me to determine if their hardware was also able to detect animal licks for my setup. The same technology is already implemented in some lick-detectors but have high financial cost. Additionally, the Delft community introduced me to a preliminary method for replacing the microstimulation setup’s most expensive component: the micromanipulator. The [Open Hardware Automated microscope](https://www.tudelft.nl/open-hardware/projects/automated-microscope) uses a lower-cost alternative to manipulate a platform, which could be adapted for microstimulation. By making hardware designs and documentation available to a wider audience, open hardware can foster a community of developers and enthusiasts who can work together to improve and enhance hardware systems.
```

File: content/blog/2023-07-28-three-months-in-delft-two-open-hardware-projects/index.en.md (hunk 2 above)
Comment (jcolomb) [reply content/blog/2023-07-28-three-months-in-delft-two-open-hardware-projects/index.en.md:109]:
```suggestion
In November 2021, I started his master project that would use the microstimulation setup. However, the setup was no longer available in the lab. The people who worked on that paradigm had left the lab, too. Thus, I set out to re-establish the setup. My only source of information was an incomplete wiring diagram. Luckily, Moritz Drüke knew a lot about the setup and could help me. With his help, I was able to get the basics of the setup to work: stimulate an electrode and detect licks. Next, I implemented the behavioural aquisition, meaning that I set up a camera that were able to clearly capture individual whisker movement and a camera that clearly captured the pupil of the animal. 
```

File: content/blog/2023-07-28-three-months-in-delft-two-open-hardware-projects/index.en.md [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+
+During this time, Dr. Julien Colomb from the Open.Make team asked me if we could work on making a good documentation of the setup. This would entail a documentation of the hardware, description of the protocol, redesigning the analysis software and getting away from large pay-walls.
+As I already was keeping track of the hardware and writing analysis, I gladly agreed.
+
+At the end of my master, I joined the open make project and went to Delft and continue to work on the microstimulation setup and its documentation. The goal was to build 2 setups (I would build one there, Jelte de Vries, another student, would build one in Berlin).
+
+## Deflt time
```
[the remark this answers is not in the document]
Comment (jcolomb) [reply content/blog/2023-07-28-three-months-in-delft-two-open-hardware-projects/index.en.md:116]:
```suggestion
## Delft time
```

File: content/blog/2023-07-28-three-months-in-delft-two-open-hardware-projects/index.en.md [hunk 5]
```diff
@@ -0,0 +1,184 @@
...
+
+
+
+# Successes and failures
+**Success :**
+Making the microstimulation project open source with regards to both soft- and hardware has already facilitated information transfer between colleagues. For instance, the build-instructions have been utilized to re-build the setup at the lab in my absence. Additionally, information is currently helping a collaboration with [the Gilad lab](https://medicine.ekmd.huji.ac.il/en/research/arielgi/Pages/aboutTheLab.aspx) located in Jerusalem. Finally, our analysis software and accompanying explaination has been used by students in the lab. 
+
+**Fails :**
+In the past we have had problems with the ordering of components. This led to long delays which, in turn, caused us to be less productive during a colaboration with the [Delft Open Hardware Academy](https://www.openhardware.academy/01_Welcome.html). We also have have lost some time while trying to develop the BPOD laboratory interface in Python. [PyBPOD](https://pybpod.readthedocs.io/projects/pybpod-api/en/v1.8.1/) is a suitable product, yet I would discourage it's use for new users.
+
```
Comment (jcolomb) [inline content/blog/2023-07-28-three-months-in-delft-two-open-hardware-projects/index.en.md:182]: ## I am a better researcher

The fellowship has been instrumental in shaping my career profile and career path. Through hands-on experience, I acquired valuable skills in soldering, PCB design, 3D printing, and project management. These technical skills have enhanced my competence as a scientist and are actively being put to use. For instance, I have recently designed a PCB layout for microstimulation; the Arduino configures the waveform and soldered wires send this to the stimulus isolator with everything nicely housed in its 3D-printed ‘containers’.  Moreover, the exposure to project management has honed my ability to plan, organize, and execute tasks efficiently. Overall, the fellowship has made me into a more well-rounded researcher and has feeling well-prepared for the coming years of my PhD.

Comment (jcolomb) [inline content/blog/2023-07-28-three-months-in-delft-two-open-hardware-projects/index.en.md:182]:
```suggestion
## I am a better researcher

The fellowship has been instrumental in shaping my career profile and career path. Through hands-on experience, I acquired valuable skills in soldering, PCB design, 3D printing, and project management. These technical skills have enhanced my competence as a scientist and are actively being put to use. For instance, I have recently designed a PCB layout for microstimulation; the Arduino configures the waveform and soldered wires send this to the stimulus isolator with everything nicely housed in its 3D-printed ‘containers’.  Moreover, the exposure to project management has honed my ability to plan, organize, and execute tasks efficiently. Overall, the fellowship has made me into a more well-rounded researcher and has feeling well-prepared for the coming years of my PhD.
```

File: content/blog/2023-07-28-three-months-in-delft-two-open-hardware-projects/index.en.md [hunk 6]
```diff
@@ -0,0 +1,184 @@
...
+* Core development team size: 2
+
+## Hardware products
+The Microstimulation setup is a customisable, open-source experimental setup that can be used for neuroscientific experiments. It uses a micromanipulator, thungsten electrode, stimulus isolator, piezo-sensor, two high-speed cameras and a BPOD.
+
+## Hardware maturity
+The setup is roughly 80% of the intended product, for use in our lab. With regards to performance the setup is ready for use. We are adapting the BPOD protocol for online visualisation of results and implementing neurophysiological recordings with multiple-channel probes.
+
+## Rebuilds 
+This project was indeed a rebuilt project from an older setup
```
[the remark this answers is not in the document]
Comment (jcolomb) [reply content/blog/2023-07-28-three-months-in-delft-two-open-hardware-projects/index.en.md:51]:
```suggestion
This project was indeed a rebuilt project from an older setup, while we also modified it quite extensively as it is using a different software
```

File: content/blog/2023-07-28-three-months-in-delft-two-open-hardware-projects/index.en.md [hunk 7]
```diff
@@ -0,0 +1,184 @@
...
+
+
+# Successes and failures
+**Success :**
+Making the microstimulation project open source with regards to both soft- and hardware has already facilitated information transfer between colleagues. For instance, the build-instructions have been utilized to re-build the setup at the lab in my absence. Additionally, information is currently helping a collaboration with [the Gilad lab](https://medicine.ekmd.huji.ac.il/en/research/arielgi/Pages/aboutTheLab.aspx) located in Jerusalem. Finally, our analysis software and accompanying explaination has been used by students in the lab. 
+
+**Fails :**
+In the past we have had problems with the ordering of components. This led to long delays which, in turn, caused us to be less productive during a colaboration with the [Delft Open Hardware Academy](https://www.openhardware.academy/01_Welcome.html). We also have have lost some time while trying to develop the BPOD laboratory interface in Python. [PyBPOD](https://pybpod.readthedocs.io/projects/pybpod-api/en/v1.8.1/) is a suitable product, yet I would discourage it's use for new users.
+
+# Aknowledgements
```
Comment (mik-schutte) [inline content/blog/2023-07-28-three-months-in-delft-two-open-hardware-projects/index.en.md:183]: Acknowledgements*

Comment (jcolomb) [reply content/blog/2023-07-28-three-months-in-delft-two-open-hardware-projects/index.en.md:183]:
```suggestion
# Acknowledgements
```

**Pull request merged**